library.md

```diff
@@ -0,0 +1,98 @@
+# Papers
+
+
+### Graph Embedding Techniques, Applications, and Performance: A Survey
+
+https://arxiv.org/abs/1705.02801v2
+
+Details:
+* This is a survey of different approaches for embeddings graphs/nodes.
+* In a section about node classification they state, that there're two approaches for classification: feature based (features based on network statistics) and random walk based (sample sequences of nodes).
+
+Thoughts: Good source of papers and some interesting comparisons of different algorithms.
+
+
+### Node2Vec
+
+http://snap.stanford.edu/node2vec/
+
+Details:
+* This is a graph-variant of word2vec model.
+* The goal is to optimize $\max_f \sum_{u\in V} \log{Pr(N_S(u)|f(u))}$, where $N_S(u)$ - the network neighborhood of node u, f - feature representation.
```

Comment:
fix equation - it's not shown properly:
`The goal is to optimize $\max_f \sum_{u\in V} \log{Pr(N_S(u)|f(u))}$, where $N_S(u)$ - the network neighborhood of node u, f - feature representation.`